Cargo.toml

```diff
@@ -122,7 +122,7 @@ cop_codegen = { path = "components/cop_codegen" }
 tipb_helper = { path = "components/tipb_helper" }
 panic_hook = { path = "components/panic_hook" }
 tipb = { git = "https://github.com/pingcap/tipb.git" }
-kvproto = { git = "https://github.com/pingcap/kvproto.git" }
+kvproto = { git = "https://github.com/pingcap/kvproto.git", branch = "release-3.0" }
```

Comment:
em, I think it is weird to depend on a release branch here. Master should depend on kvproto master.

Reply:
We will do a fast-forward merge a few days later and after that I will update to master.

Reply:
can you add a comment here?